architecture/core_concepts/routes.adoc

```diff
@@ -1052,3 +1053,66 @@ spec:
     in the subdomain `example.com`. `*.example.com` is the subdomain for host
     name `wildcard.example.com` to reach the exposed service.
 ====
+
+[[disable-namespace-ownership-check]]
+== Disabling Namespace Ownership Check
+
+By default, different paths to a host name (and subdomains for wildcard routes)
+can only be claimed by routes in the same namespace as the owning route.
+
+For example, if namespace ns1 creates the oldest route r1  www.abc.xyz, then it
+owns that host name and subdomain (abc.xyz) for wildcard routes.  If another
+namespace ns2 tries to create a route with say a different path www.abc.xyz/path1/path2,
+it would fail because a route in another namespace (ns1 in this case) "owns"
+that host.
+
+It becomes more complex with
+xref:../../install_config/router/default_haproxy_router.html#using-wildcard-routes[wildcard routes]
+as we now need to look at routes that are granted claims to hosts (*.abc.xyz)
+that belong to the subdomain (abc.xyz) associated with the wildcard route.
+
+The disable namespace ownership check relaxes those restrictions and allows
+hosts (and subdomains) to be claimed across namespaces.
+
+[WARNING]
+====
+This feature can compromise security since it can allow any namespace to be
+granted claims to hosts that overlap (either wildcard subdomains or paths for
+a regular host). When this is used, make sure your cluster policy has locked
+down route creation by end users.
```

Comment:
@ramr I am not sure how to word this. Could you suggest a wording? I am not sure we want to suggest a dev use (devs will figure this out).